go.mod

```diff
@@ -130,6 +130,8 @@ require (
 	modernc.org/sqlite v1.20.3
 )
 
+require github.com/hamba/avro/v2 v2.4.0
```

Comment:
Please move this import into the require block above. I know that the go tooling did this, but please clean it up :)

Reply:
Done